### PR TITLE
Update the SDK version

### PR DESCRIPTION
**Pull request by krzysztofwolski**

### CODE REVIEW

**Review by lkostrowski — COMMENTED** [review]
if it works, lets release new app-sdk and bump to `@latest` then

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "@material-ui/core": "^4.12.4",
     "@material-ui/icons": "^4.11.3",
     "@material-ui/lab": "4.0.0-alpha.61",
-    "@saleor/app-sdk": "0.24.0-0",
+    "@saleor/app-sdk": "0.26.0-4",
```
Comment (lkostrowski) [inline package.json:20]: lets set proper tag

File: package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
     "graphql": "^16.6.0",
     "graphql-tag": "^2.12.6",
     "instantsearch.css": "^7.4.5",
-    "next": "13.0.2",
+    "next": "13.1.2",
```
Comment (lkostrowski) [inline package.json:29]: probably you should also update eslint-config-next

File: saleor-app.ts [hunk 3]
```diff
@@ -24,16 +24,14 @@ switch (aplType) {
     apl = new FileAPL();
 
     break;
-  case "rest": {
+  case "cloud": {
```
Comment (lkostrowski) [inline saleor-app.ts:27]: remember to update deployment env

File: src/components/AlgoliaConfigurationCard.tsx [hunk 4]
```diff
@@ -22,16 +22,16 @@ export const AlgoliaConfigurationCard = () => {
...
+    queryFn: async () => fetchConfiguration(saleorApiUrl!, token!),
+    enabled: !!token && !!saleorApiUrl,
   });
 
   const { mutate, isLoading: isMutationLoading } = useMutation(
     async (conf: AlgoliaConfigurationFields) => {
       const resp = await fetch("/api/configuration", {
         method: "POST",
         headers: {
-          "saleor-domain": domain!,
```
Comment (lkostrowski) [inline src/components/AlgoliaConfigurationCard.tsx:34]: lets import header names from app-sdk

**Review by lkostrowski — CHANGES_REQUESTED** [review]

**Comment by krzysztofwolski** [comment]
APL variable name updated to `saleor-cloud` in deploymed app

**Review by lkostrowski — APPROVED** [review]

**Pull request merged**